### PR TITLE
Add informational messages for app expansion

### PR DESCRIPTION
**Pull request by brenthdaniel**
PR for #7325

### CODE REVIEW

**Comment by brenthdaniel** [comment]
#build

**Comment by brenthdaniel** [comment]
Ugh, missed checking in a file... 
#build again

File: dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops [hunk 1]
```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
 error.file.outside.archive=CWWKZ0132W: The file {0} in the archive {1} will not be extracted because the path is not contained within the archive.
 error.file.outside.archive.explanation=The file's path specifies a relative directory. Extracting the file would result in the file being located outside of the archive's root directory.
 error.file.outside.archive.useraction=Remove any files with relative paths from the archive.  
+
+#{0} is the application name, {1} is the location of the EAR or WAR file, {2} is the location where the file is expanded
+info.expanding.app=CWWKZ0133I: The application named {0} at location {1} is being expanded to the directory {2}.
```
Comment (dmuelle) [inline dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops]: **change**
The application named {0} at location {1} is being expanded to the directory {2}.
**to**
The {0} application at the {1} location is being expanded to the {2} directory.

File: dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops [hunk 2]
```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
...
+info.expanding.app.explanation=The archive file for the application is being expanded because automatic expansion of applications was configured.
+info.expanding.app.useraction=No action is required.
+
+info.loose.app=CWWKZ0134I: The application named {0} is using the loose application definition at location {1}.
+info.loose.app.explanation=The location for the application points to an XML file that defines a loose application. 
+info.loose.app.useraction=No action is required.
```
Comment (dmuelle) [inline dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops]: ~~No action is required.~~
For informational messages, leave the useraction filed blank.

File: dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops [hunk 3]
```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
...
+
+info.directory.app=CWWKZ0135I: The application named {0} is using the expanded directory at location {1}.
+info.directory.app.explanation=The location for the application points to a directory rather than an archive file. 
+info.directory.app.useraction=No action is required.
```
Comment (dmuelle) [inline dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops]: ~~No action is required.~~
For informational messages, leave the useraction filed blank.

File: dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops [hunk 4]
```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
...
+
+info.unexpanded.app=CWWKZ0136I: The application named {0} is using the archive file at location {1}.
+info.unexpanded.app.explanation=The location for the application points to a binary archive file. The archive file will not be expanded on disk because automatic expansion of applications was not configured.
+info.unexpanded.app.useraction=No user action is required.
```
Comment (dmuelle) [inline dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops]: ~~No action is required.~~
For informational messages, leave the useraction filed blank.

File: dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops [hunk 5]
```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
 error.file.outside.archive=CWWKZ0132W: The file {0} in the archive {1} will not be extracted because the path is not contained within the archive.
 error.file.outside.archive.explanation=The file's path specifies a relative directory. Extracting the file would result in the file being located outside of the archive's root directory.
 error.file.outside.archive.useraction=Remove any files with relative paths from the archive.  
+
+#{0} is the application name, {1} is the location of the EAR or WAR file, {2} is the location where the file is expanded
+info.expanding.app=CWWKZ0133I: The application named {0} at location {1} is being expanded to the directory {2}.
+info.expanding.app.explanation=The archive file for the application is being expanded because automatic expansion of applications was configured.
+info.expanding.app.useraction=No action is required.
+
+info.loose.app=CWWKZ0134I: The application named {0} is using the loose application definition at location {1}.
```
Comment (dmuelle) [inline dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops]: **change**
The application named {0} is using the loose application definition at location {1}.
**to**
The {0} application  is using the loose application definition at the {1} location.

File: dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops [hunk 6]
```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
 error.file.outside.archive=CWWKZ0132W: The file {0} in the archive {1} will not be extracted because the path is not contained within the archive.
 error.file.outside.archive.explanation=The file's path specifies a relative directory. Extracting the file would result in the file being located outside of the archive's root directory.
 error.file.outside.archive.useraction=Remove any files with relative paths from the archive.  
+
+#{0} is the application name, {1} is the location of the EAR or WAR file, {2} is the location where the file is expanded
+info.expanding.app=CWWKZ0133I: The application named {0} at location {1} is being expanded to the directory {2}.
+info.expanding.app.explanation=The archive file for the application is being expanded because automatic expansion of applications was configured.
+info.expanding.app.useraction=No action is required.
```
Comment (dmuelle) [inline dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops]: ~~No action is required.~~
For informational messages, leave the useraction filed blank.

File: dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops [hunk 7]
```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
...
+
+info.loose.app=CWWKZ0134I: The application named {0} is using the loose application definition at location {1}.
+info.loose.app.explanation=The location for the application points to an XML file that defines a loose application. 
+info.loose.app.useraction=No action is required.
+
+info.directory.app=CWWKZ0135I: The application named {0} is using the expanded directory at location {1}.
```
Comment (dmuelle) [inline dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops]: **change**
The application named {0} is using the expanded directory at location {1}.
**to**
The {0} application is using the expanded directory at the {1} location.

File: dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops [hunk 8]
```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
...
+info.directory.app.explanation=The location for the application points to a directory rather than an archive file. 
+info.directory.app.useraction=No action is required.
+
+info.unexpanded.app=CWWKZ0136I: The application named {0} is using the archive file at location {1}.
```
Comment (dmuelle) [inline dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops]: **change**
The application named {0} is using the archive file at location {1}.
**to**
The {0} application is using the archive file at the {1} location.

**Comment by brenthdaniel** [comment]
@dmuelle Thanks, the updates have been made.

**Pull request merged**